### ISSUE TITLE
[Techstack-beta] Blog: Filter UX keyboard navigation

### ISSUE THREAD

**Issue by samueldr** [NixOS/nixos-homepage#1290]
I will frontload with my opinion that this should be moved back to discrete pages. Even with bugs fixed, I do not believe this is the correct concept to use here.

(It also avoids many issues by doing nothing instead...)

* * *

See #1248.

This was not fixed in #1265.

* * *

There is no way to activate these items using keyboard navigation.

Though, as is usual with accessibility things, things get awkward and hard quickly.

![image](https://github.com/NixOS/nixos-homepage/assets/132835/f362f4ee-6d42-4e6f-9ddc-118b75592929)

In this screenshot, we can see that Vimium see the item! This is likely coming from the use of `<label>` here.

But as can be observed in the accessibility tree, it cannot be focused, which is why keyboard navigation cannot happen here.

The underlying element powering the widget is an `<input>`, but since it is hidden, it is not part of the tree.
